styles.css

```diff
@@ -0,0 +1,76 @@
+*{
+    box-sizing:border-box;
+    margin:0;
+    padding:0;
```

Comment:
Try to leave a space before the value for your css property like `box-sizing: border-box;`